FestivalPlanner/templates/films/results.html

```diff
@@ -19,7 +19,7 @@ <h2 class="row col-left-h2"><a href="{{ film.url }}">{{ film.title }}</a></h2>
     <ul>
         <li>Film#: {{ film.film_id }}</li>
         <li>Duration: {{ film.duration }}</li>
-        <li>Subsection: {{ subsection.name }}</li>
+        <li>Subsection: {% if film.subsection %} {{ film.subsection.name }} {% else %} - {% endif %}</li>
         <li>Description: {{ description }}</li>
         <li>Reviewer: {% if film.reviewer %} {{ film.reviewer }} {% else %} - {% endif %}</li>
```

Comment:
Please consider to define the representation of an empty `description` in HTML too.

Reply:
Done.